Delivery-service/Models/Contracts/ICompany.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using Models.Models;
+
```

Comment:
wich project main ?
"Delivery-service" or "Models"
i think main should be as console app
another - class library project type